addons/website_sale/views/product_product_add.xml

```diff
@@ -20,6 +23,7 @@
         <field name="res_model">product.product</field>
         <field name="view_mode">form</field>
         <field name="target">new</field>
+        <field name="context" eval="{'dialog_size': 'medium', 'isEcommerce': True}"/>
```

Comment:
```suggestion
        <field name="context" eval="{'dialog_size': 'medium'}"/>
```

Reply:
Now, the specs changed again and we have to publish the product created from this form view ( from pos or barcode module ) only when the website_category is set, and if the product is created from the website_sale module we will always publish it directly.

So the reason to pass this context is to find from where this product is created or updated if we get it from any of the other module we won't get this context and will be easier to identify from where this is called.

As this view is primary we will just get this context when created from the website_sale module

Reply:
Please use a clear dedicated key then, maybe something like `publish_product` ?
`isEcommerce` is too vague and we don't know what it's used for.

Reply:
I have changed it as per the suggestion.
Please check it.